driver/registry_base.go

```diff
@@ -552,3 +553,20 @@ func (m *RegistryBase) HSMContext() hsm.Context {
 func (m *RegistrySQL) ClientAuthenticator() x.ClientAuthenticator {
 	return m.OAuth2Provider().(*fosite.Fosite)
 }
+
+func (m *RegistryBase) LogoutIdentity(ctx context.Context, identity string) error {
+	kratosURL, ok := m.conf.KratosAdminURL(ctx)
+	if !ok {
+		return nil
+	}
+	client := kratosClient.NewAPIClient(&kratosClient.Configuration{
+		Host:       kratosURL.Host,
+		Scheme:     kratosURL.Scheme,
+		UserAgent:  fmt.Sprintf("Ory Hydra version %s", m.BuildVersion()),
+		Servers:    kratosClient.ServerConfigurations{{URL: kratosURL.String()}},
+		HTTPClient: retryablehttp.NewClient().StandardClient(),
+	})
+	_, err := client.IdentityApi.DeleteIdentitySessions(ctx, identity).Execute()
```

Comment:
I'm not sure which API we want to call here.

For user-initiated logout, do we just want to terminate the current session, or all sessions?

For a backchannel logout, we don't have a current user session, so I guess we want to terminate all user sessions?